lib/server/readMetadata.js

```diff
@@ -271,7 +273,7 @@ function generateMetadataDocs() {
     const extension = path.extname(file);
 
     if (extension === '.md' || extension === '.markdown') {
-      const res = processMetadata(file);
+      const res = processMetadata(file, language);
```

Comment:
Don't you want to pass `language` in to the other calls to `processMetadata`?

e.g., on line 219 in this file:

`const res = processMetadata(file);`

Reply:
We only need to specify the language case, if we use `translated_docs`. In general, we do not need to specify, because we use `getDocsPath`. It seems to increase unnecessary processing.

Of course I know a lot more bugs related to this project's multilingual, and I was fixed it. However, there are many cases where we need to change the structure a lot, so I want to send PR in gradually.

Reply:
Hi @gimdongwoo - In general though, I don't like leaving a parameter missing to a function call. It can cause confusion in reading the code. Even if you don't pass `language` -- which I still think is ok - can you pass something, even if a `''`?